protected-api-access-protected-api/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "@azure/msal-node": "^1.4.0",
+    "@azure/msal-node-extensions": "^1.0.0-alpha.13",
```

Comment:
You don't use the extensions, right?  Should just need `@azure/msal-node` from what I saw.  But maybe I missed it?

Reply:
You are correct

Reply:
🪓